spec/unit/govuk_index/indexable_content_sanitiser_spec.rb

```diff
@@ -1,70 +1,72 @@
 require 'spec_helper'
 
 RSpec.describe GovukIndex::IndexableContentSanitiser do
-  it "strips_html_tags_from_indexable_content" do
-    payload = [
-      [
-        { "content_type" => "text/govspeak", "content" => "**hello**" },
-        { "content_type" => "text/html", "content" => "<strong>hello</strong> <a href='www.gov.uk'>marmaduke</a>" }
-      ]
-    ]
-
-    expect(subject.clean(payload)).to eq("hello marmaduke")
-  end
-
-  it "passes_single_string_content_unchanged" do
+  it "passes content without html tags unchanged" do
     payload = ["hello marmaduke"]
 
     expect(subject.clean(payload)).to eq("hello marmaduke")
   end
 
-  it "passes_multiple_string_items_unchanged" do
+  it "can process multiple string items" do
     payload = ["hello marmaduke", "hello marley"]
 
     expect(subject.clean(payload)).to eq("hello marmaduke\n\n\nhello marley")
   end
 
-  it "strips_html_tags_from_string_content" do
+  it "strips html tags from string content" do
     payload = ["<h1>hello marmaduke</h1>"]
 
     expect(subject.clean(payload)).to eq("hello marmaduke")
   end
 
-  it "multiple_html_text_payload_items" do
-    payload = [
-      [
-        { "content_type" => "text/govspeak", "content" => "**hello**" },
-        { "content_type" => "text/html", "content" => "<strong>hello</strong>" }
-      ],
-      [
-        { "content_type" => "text/govspeak", "content" => "**goodbye**" },
-        { "content_type" => "text/html", "content" => "<strong>goodbye</strong>" }
-      ],
-    ]
+  context "when html test payloads exists" do
+    it "strips out html tags from to the html content" do
+      payload = [
+        [
+          { "content_type" => "text/govspeak", "content" => "**hello**" },
+          { "content_type" => "text/html", "content" => "<strong>hello</strong> <a href='www.gov.uk'>marmaduke</a>" }
+        ]
+      ]
+
+      expect(subject.clean(payload)).to eq("hello marmaduke")
+    end
+
+    it "correctly processes multiple payload items" do
```

Comment:
"correctly" makes it hard to tell if the test is looking for the right thing. Can we rename it to describe the behaviour better? Like "it joins multiple payload items with newlines"?